reference/api.md

```diff
@@ -0,0 +1,10 @@
+---
+title: API Links
+description: External links to PlaceOS Apiary
+---
+
+For detailed information on PlaceOS API, please see them on Apiary:
```

Comment:
The fact these are on Apiary is not important information.

Reply:
Agreed @mghill this can be removed.